documentation/writing_plans.md

```diff
@@ -692,12 +692,12 @@ to run on the targets. You must configure the PuppetDB client before you run it.
 ```
 plan run_with_facts(TargetSpec $targets) {
   # This collects facts on targets and update the inventory
-  run_plan(**puppetdb_fact**, targets => $targets)
```

Comment:
Wouldnt this be the only thing that errored?

Reply:
The single quotes all over dont seem necessary?

Reply:
Huh, TIL you don't need to quote task/plan names and parameters.
I do think having the consistency is preferable, though. I'll update the commit message.

Reply:
Consistency is good. Names like `'my_module::my_task'` require quotes.